### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This fixes the test runner incompatibility with Microsoft.AspNetCore.App. Thanks for figuring this out guys!

### CODE REVIEW

File: src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj [hunk 1]
```diff
@@ -6,9 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```
Comment (JamesNK) [inline src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj]: @natemcmaster Is `FrameworkReference` prefered over `ProjectReference` in class libraries?

Comment (JunTaoLuo) [reply src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj]: If we want to keep the `PackageReference`, I assume we'll want to mark it with `PrivateAssets`

Comment (natemcmaster) [reply src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj]: >  Is FrameworkReference prefered over ProjectReference in class libraries?

ProjectReference to what? Microsoft.AspNetCore.App isn't a project in this repo.

> If we want to keep the PackageReference, would we want to mark it with PrivateAssets?

By design, we really don't want you to use Microsoft.AspNetCore.App 3.0 as a package, so this is the right move.

**Comment by JunTaoLuo** [comment]
// @jtattermusch

File: src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj (hunk 1 above)
Comment (JamesNK) [reply src/Grpc.AspNetCore.Server/Grpc.AspNetCore.Server.csproj]: > ProjectReference to what? Microsoft.AspNetCore.App isn't a project in this repo.

Opps, I meant PackageReference 😐

**Pull request merged**